_episodes/04-quality-control.md

```diff
@@ -102,31 +102,25 @@ document the manipulations or analyses that were carried out on those data.
 [Cornell University's Research Data Management Service Group](https://data.research.cornell.edu/content/readme) provides detailed
 guidelines for how to write a good readMe file, along with an adaptable template.
 
-<!-- [Example: converting all data to values: use soybean aphid suction trap dataset for this section] -->
-
 ## Sorting
 Bad values often sort to the bottom or top of the column. For example, if your data should be numeric, then alphabetical and null data
 will group at the ends of the sorted data. Sort your data by each field, one at a time. Scan through each column, but pay the most
 attention to the top and the bottom of a column. 
 If your dataset is well-structured and does not contain formulas, sorting should never affect the integrity of your dataset.
 
-**Remember** to expand your sort in order to prevent data corruption. Expanding your sort ensures that the all the data in one row move together instead of only sorting a single column in isolation as shown in the screenshot.
-
-![Figure of Sorting menu](../fig/sorting.png)
-
-Let's try the following sorting exercise:
+**Remember** to expand your sort in order to prevent data corruption. Expanding your sort ensures that the all the data in one row move together instead of only sorting a single column in isolation. Sorting by only a single column will scramble your data - a single row will no longer represent an individual observation.
 
 > ## Exercise : 
 >
-> Download this data file based on the 2013 survey data to your computer: [survey_sorting_exercise](https://ndownloader.figshare.com/files/2252083)
+> We've combined all of the tables from the messy data into a single table in a single tab. Download this semi-cleaned data file to your computer: [survey_sorting_exercise](https://github.com/datacarpentry/spreadsheet-ecology-lesson/blob/gh-pages/data/survey_data_messy_quality_control.xlsx)  (click `Download`)
```

Comment:
Going to that link won't download it by default. Link should be https://github.com/datacarpentry/spreadsheet-ecology-lesson/blob/gh-pages/data/survey_data_messy_quality_control.xlsx?raw=true